torch/fx/experimental/proxy_tensor.py

```diff
@@ -303,6 +304,20 @@ def __torch_dispatch__(cls, func_overload, types, args=(), kwargs=None):
                     break
         assert proxy_mode is not None, "At least one argument must be a ProxyTensor"
 
+        if func_overload.__name__ == 'torch.cond':
+            # TODO: need to parse
+            assert kwargs is None or not kwargs
+            pred, true_fn, false_fn, *operands = args
+
+            g_true_fn = get_isolated_graphmodule(true_fn, *operands, {})
+            g_false_fn = get_isolated_graphmodule(false_fn, *operands, {})
```

Comment:
So it did work in the end? What was the problem originally?

Reply:
Discussed offline.